superset-frontend/src/views/CRUD/welcome/DashboardTable.tsx

```diff
@@ -149,9 +148,9 @@ function DashboardTable({
         buttons={[
           {
             name: (
-              <IconContainer>
-                <Icon name="plus-small" /> Dashboard{' '}
-              </IconContainer>
+              <div>
+                <i className="fa fa-plus" /> Dashboard{' '}
```

Comment:
Same here... would prefer to use our custom icons if possible, but those are being revamped presently

Reply:
Same as previous comment.